src/util/strings.ts

```diff
@@ -50,6 +50,39 @@ export const getIncrementRatingButtonLabel = (
 	}'s ${attribute} from ${interimRating} to ${increment ? nextRatingAbove : nextRatingBelow}`;
 };
 
+/**
+ * Returns the string needed for the props in the PopUp component in the Submit Rating page
+ * @param attribute
+ * @param propsType
+ * @returns
+ */
+export const getPopUpProps = (
+	attribute: string,
+	propsType:
+		| "buttonAccessibility"
+		| "text"
+		| "modalAccessibility"
+		| "closeButtonAccessibilityLabel"
+): string => {
+	let displayString!: string;
+
+	switch (propsType) {
+		case "buttonAccessibility":
+			displayString = `A button that triggers a pop up to open. This will explain what ${attribute} is.`;
+			break;
+		case "modalAccessibility":
+			displayString = `A pop up that explains ${attribute}`;
+			break;
+		case "closeButtonAccessibilityLabel":
+			displayString = `Close ${attribute} pop up`;
```

Comment:
Listening to this, i think it's too wordy - they can only have one popup open at-a-time anyway.
Just "close this pop up" is probably good

Reply:
That's a good point